.readthedocs.yml

```diff
@@ -9,6 +9,22 @@ build:
   os: "ubuntu-22.04"
   tools:
     python: "3.10"
+  jobs:
```

Comment:
@romilbhardwaj we should get this in first, to avoid our readthedoc build every PR.